packages/e2e-tests/specs/editor/various/switch-to-draft.test.js

```diff
@@ -137,6 +135,11 @@ describe( 'Clicking "Switch to draft" on a published post/page', () => {
 					expect( postStatus ).toBe( 'publish' );
 				} );
 				it( `should revert a published ${ postType } to a draft if confirmed`, async () => {
+					// Capitalize postType for use in snackbar XPath
+					const capitalizedPostType =
```

Comment:
We might also be able to use the [`lower-case()`](https://www.w3.org/TR/xpath-functions/#func-lower-case) or [`matches()`](https://www.w3.org/TR/xpath-functions/#func-matches) functions within the XPath selector.

Reply:
Those are both new to me, thanks for the links @aaronrobertshaw! I wasn't able to implement either, even when testing directly in Chrome's dev tools with strings I _know_ should match it didn't recognize the elements I was targeting. Not sure if it's user error on my part (likely) or something else, but for now I've gone with @ciampo's `toLowerCase()` suggestion.

I don't _love_ everything about this implementation... for one thing we're locating the snackbar notification by finding just the word "published" or "reverted" respectively, then pulling the text, lower casing it, and comparing it a full string to my current post type. Feels kind of like taking the long way around, and almost makes me wonder if I shouldn't just stop at identifying a snackbar notif with published/reverted inside it 🤔

Reply:
>  Not sure if it's user error on my part (likely) or something else,

Maybe it's related to the version of XPath (it seems like those functions are available from version 2.0)?

> I don't love everything about this implementation... for one thing we're locating the snackbar notification by finding just the word "published" or "reverted" respectively, then pulling the text, lower casing it, and comparing it a full string to my current post type. Feels kind of like taking the long way around, and almost makes me wonder if I shouldn't just stop at identifying a snackbar notif with published/reverted inside it 🤔

It does seem convoluted, but I don't know if there's a better way to test for a confirmation that the post was published / reverted. Let's keep it as-is for now, and iterate on it in case we find a better solution!